cmd/dp-import-tracker/main.go

```diff
@@ -62,34 +64,34 @@ func (trackedInstances trackedInstanceList) updateInstanceFromImportAPI(api *api
 		tempCopy.observationsInsertedCount = observationsInsertedCount
 	}
 	// update jobID (should only 'change' once, on first update after initial creation of the instance - which has no jobID)
-	if importID := instanceFromAPI.Job.ID; importID != trackedInstances[instanceID].jobID {
+	if importID := instanceFromAPI.Links.Job.ID; importID != trackedInstances[instanceID].jobID {
 		tempCopy.jobID = importID
 	}
 	trackedInstances[instanceID] = tempCopy
 	return nil
 }
 
 // getInstanceListFromImportAPI gets a list of current import Instances, to seed our in-memory list
-func (trackedInstances trackedInstanceList) getInstanceListFromImportAPI(api *api.ImportAPI) error {
-	instancesFromAPI, err := api.GetInstances(url.Values{"instance_states": []string{"created"}})
+func (trackedInstances trackedInstanceList) getInstanceListFromImportAPI(datasetAPI *api.DatasetAPI) error {
```

Comment:
Function name still says import. 

Also probably not necessary to rename the var from api to datasetAPI though, as the function definition provides context as to what type the api is. Not a major issue, just idiomatic to tend towards shorter var names where reasonable.

Reply:
I have taken the API name out of the function name as it did not seem necessary. I have also renamed datasetAPI to api as suggested.